items/src/test/scala/weco/api/items/ItemsApiFeatureTest.scala

```diff
@@ -36,42 +27,30 @@ class ItemsApiFeatureTest
     with ItemsApiFixture
     with JsonAssertions
     with IntegrationPatience
-    with WorkGenerators
-    with ItemsGenerators {
+    with ItemsApiGenerators {
 
   describe("look up the status of an item") {
     it("shows a user the items on a work") {
-      val item = createIdentifiedItemWith(
-        sourceIdentifier = SourceIdentifier(
-          identifierType = SierraSystemNumber,
-          value = "i16010176",
-          ontologyType = "Item"
-        )
+      val sierraItemNumber = createSierraItemNumber
+
+      val temporarilyUnavailableOnline = AccessCondition(
+        method = AccessMethod.OnlineRequest,
+        status = AccessStatus.TemporarilyUnavailable
+      )
+
+      val physicalItem = createPhysicalItemWith(
+        sierraItemNumber = sierraItemNumber,
+        accessCondition = temporarilyUnavailableOnline
       )
 
-      val work = indexedWork().items(List(item))
+      val work = indexedWork().items(List(physicalItem))
 
       val responses = Seq(
         (
-          HttpRequest(
-            uri = Uri(
-              "http://sierra:1234/v5/items/1601017?fields=deleted,fixedFields,holdCount,suppressed"
-            )
-          ),
+          HttpRequest(uri = sierraUri(sierraItemNumber)),
           HttpResponse(
-            entity = HttpEntity(
-              contentType = ContentTypes.`application/json`,
-              """
-                |{
-                |  "id": "1601017",
-                |  "deleted": false,
-                |  "suppressed": false,
-                |  "fixedFields": {
-                |    "88": {"label": "STATUS", "value": "-", "display": "Available"}
-                |  },
-                |  "holdCount": 0
-                |}
-                |""".stripMargin
+            entity = sierraItemResponse(
```

Comment:
Could this be named something to emphasise that it's a response that make the item available to request online? It'd make this test clearer – right now you have to go look elsewhere to see what's going on here.

(Or leave the request body here, which IMO was fine.)